conf/repos.yaml.template

```diff
@@ -35,3 +35,290 @@ REPOS:
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+
+  YUM_0:
+    URL:
+    ERRATA:
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+
+  YUM_1:
+    URL:
+    ERRATA:
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+
+  YUM_2:
+    URL:
+    ERRATA:
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+
+  YUM_3:
+    URL:
+    ERRATA:
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+
+  YUM_4:
+    URL:
+    ERRATA:
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+
+  YUM_6:
+    URL:
+    ERRATA:
+      - "EX1"
+      - "EX2"
+      - "EX3"
+      - "EX4"
+      - "EX5"
+
+  YUM_7:
+    URL:
+
+  YUM_8:
+    URL:
+
+  YUM_9:
+    URL:
+    ERRATA:
+      - "EX1"
+      - "EX2"
+      - "EX3"
+      - "EX4"
+      - "EX5"
+
+  YUM_10:
+    URL:
+    ERRATA:
+      - "EX1"
+
+  YUM_11:
+    URL:
+
+  YUM_MIXED:
+    URL:
+
+  CUSTOM_PUPPET:
+    URL:
+
+  PUPPET_0:
+    URL:
+
+  PUPPET_1:
+    URL:
+
+  PUPPET_2:
+    URL:
+
+  PUPPET_3:
+    URL:
+
+  PUPPET_4:
+    URL:
+
+  PUPPET_5:
+    URL:
+
+  PUPPET_8:
+    URL:
+
+  REPO_DISCOVERY:
+    URL:
+    ERRATA:
+      - "EX1"
+      - "EX2"
+      - "EX3"
+      - "EX4"
+      - "EX5"
+      - "EX6"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
+      - "EX#"
```

Comment:
I think you can cut these short, to 2 list items max.

Reply:
Some of these need 6. Should I limit them all to that number?

Reply:
can you link which test needs these many blank entries?

Reply:
cli/test_errata.py::test_positive_user_permission
cli/test_errata.py::check_errata function
cli/test_errata.py::products_with_repos fixture
api/test_errata.py::test_positive_list
ui/test_errata.py::test_positive_list

Reply:
I'm not going to consider this blocking, but the information about which test modules require multiple entries should be added in-line to the template file.

Reply:
Ok I can add those changes.